core/src/components/select/select.ionic.scss

```diff
@@ -181,6 +182,18 @@
   color: globals.$ion-primitives-neutral-500;
 }
 
+// Focus
+// ---------------------------------------------
+:host(.ion-focused) .select-wrapper-inner {
```

Comment:
Based on the designs, there shouldn't be an offset for any of the selects. But I'm seeing an offset for the outline variant.

Reply:
Thanks for the heads-up, will change it!

Reply:
This is done ✅